user_guide_src/source/libraries/sessions/002.php

```diff
@@ -1,3 +1,3 @@
 <?php
 
-$session
+$session;
```

Comment:
The last `;` is odd. It is not syntax error, but do nothing.

```
Once loaded, the Sessions library object will be available using:
$session;
```

Revert to the RST file?

Reply:
I chose to revert this specific examples.  For others, I decided to make them functional in the same way as similar examples were already present.
See [5a685df](https://github.com/codeigniter4/CodeIgniter4/pull/5722/commits/5a685df6d89cc91ed5f3891f9daba60b46056c2a).

Reply:
Okay. Thanks!